core/src/main/java/org/mapstruct/Mapper.java

```diff
@@ -15,9 +15,35 @@
 import static org.mapstruct.NullValueCheckStrategy.ON_IMPLICIT_CONVERSION;
 
 /**
+ * //TODO should I add other examples or it's enough?
  * Marks an interface or abstract class as a mapper and activates the generation of a implementation of that type via
  * MapStruct.
  *
+ * <p>
```

Comment:
I guess it would be interesting to have some more examples here for instance on 'uses' and on 'componentModel'